envs/quadrotors/quad_multi_model.py

```diff
@@ -11,11 +11,23 @@ class QuadMultiMeanEncoder(EncoderBase):
     def __init__(self, cfg, obs_space, timing, self_obs_dim=18, neighbor_obs_dim=6, neighbor_hidden_size=32, obstacle_obs_dim=6, obstacle_hidden_size=32):
         super().__init__(cfg, timing)
         self.self_obs_dim = self_obs_dim
-        self.neighbor_obs_dim = neighbor_obs_dim
+        self.neighbor_obs_type = cfg.neighbor_obs_type
         self.neighbor_hidden_size = neighbor_hidden_size
         self.use_spectral_norm = cfg.use_spectral_norm
         self.obstacle_mode = cfg.quads_obstacle_mode
-        self.num_use_neighbor_obs = int(cfg.quads_ratio_use_neighbor_obs * (cfg.quads_num_agents - 1))
+        if cfg.quads_local_obs == -1:
+            self.num_use_neighbor_obs = cfg.quads_num_agents - 1
+        else:
+            self.num_use_neighbor_obs = cfg.quads_local_obs
```

Comment:
Maybe assert that this is a correct value.

Reply:
This is done in quadrotor_multi.py line 36